service-front/app/test/CommonTest/Middleware/ErrorHandling/GoneHandlerTest.php

```diff
@@ -8,30 +8,34 @@
 use Fig\Http\Message\StatusCodeInterface;
 use Mezzio\Template\TemplateRendererInterface;
 use PHPUnit\Framework\TestCase;
+use Prophecy\Argument;
 use Prophecy\PhpUnit\ProphecyTrait;
 use Psr\Http\Message\ResponseFactoryInterface;
 use Psr\Http\Message\ResponseInterface;
 use Psr\Http\Message\ServerRequestInterface;
 use Psr\Http\Message\StreamInterface;
+use Psr\Http\Message\UriInterface;
 use Psr\Http\Server\RequestHandlerInterface;
 
 class GoneHandlerTest extends TestCase
 {
     use ProphecyTrait;
 
-    public function testReturnsGoneResponseForGoneUris(): void
+    /**
+     * @test
+     */
```

Comment:
I believe you can use the `PHPUnit\Framework\Attributes\Test` attribute on these tests, which is a bit more declarative
```suggestion
    #[Test]
```

Reply:
Service-front hasn't been moved to PHPUnit 10 yet. I've got a rector script ready to tackle it when I find the time to sort that.